### ISSUE TITLE
I would add npm start method.

### ISSUE THREAD

**Issue by quantuminformation** [lucamezzalira/jsday-cycle-js#1]
Also index file doesn't point to dist.

**Comment by lucamezzalira** [comment]
good point, currently I'm copying index.html to dist folder so you can use any webserver pointing to dist to see the application running (livereloadx for instance) but yes it could be an improvement.
thanks for sharing your thoughts
